plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/notifications/ProcessNotificationsBase.kt

```diff
@@ -15,16 +15,20 @@ import com.intellij.openapi.project.Project
 import software.aws.toolkits.core.utils.inputStream
 import software.aws.toolkits.jetbrains.utils.notifyStickyWithData
 import java.nio.file.Paths
+import java.util.concurrent.atomic.AtomicBoolean
 
 object NotificationMapperUtil {
     val mapper = jacksonObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
 }
 
 @Service(Service.Level.PROJECT)
-class ProcessNotificationsBase {
+class ProcessNotificationsBase(
+    private val project: Project,
+) {
     private val notifListener = mutableListOf<NotifListener>()
+    private var isStartup: AtomicBoolean = AtomicBoolean(true)
```

Comment:
wouldn't you want startup notifications to propagate consistently across all projects on startup?